### DISCUSSION TITLE
there are unknown plugin options for "gatsby-plugin-sass": additionalData

### DISCUSSION THREAD

**Discussion by natapg** [gatsbyjs/gatsby#34705] [Help]
with the latest update of the gatsby-plugin-sass plugin I get this warning message `there are unknown plugin options for "gatsby-plugin-sass": additionalData` , we are using the additionalData to import a global scss like this:

```js
    {
      resolve: 'gatsby-plugin-sass',
      options: {
        implementation: require('sass'),
        additionalData: `@import "${resolvePath(__dirname, '/src/styles/common')}";`,
        cssLoaderOptions: {
          esModule: false,
          modules: {
            namedExport: false,
          },
        },
      },
    },
```

The import still works, so the warning is wrong, since the option is still valid

gatsby: 4.5.3 ->  4.6.1
gatsby-plugin-sass: 5.5.0 -> 5.6.0

**Comment by LekoArts** [comment]
This might be related: https://github.com/gatsbyjs/gatsby/pull/34682 (and the PR that introduced this change)

**Comment by EdAndrade** [comment]
Sharing this concern too, nothing changed

**Comment by arvind-v97** [comment]
Is this fixed?

**Reply by EdAndrade** [reply]
nop @arvind-v97
